org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/thingTranslator.js

```diff
@@ -56,21 +75,7 @@ class Translator {
                 return this._getEndpointByLocationId(payload.locationId).then((endpoints) => {
                     if (endpoints.length === 0) return undefined;
                     
-                    //Check that the ST endpoint is valid (actually has access to devices)
-                    var endpointPromises = [];
-                    endpoints.forEach((endpointUri) => {
```

Comment:
Yes. I updated the device change payload in the SmartApp to return the full device status, so we do not need to query for device details again.

---
In reply to: [112324063](https://github.com/openT2T/translators/pull/234#discussion_r112324063) [](ancestors = 112324063)